CONTRIBUTING_GUIDELINES.md

```diff
@@ -91,6 +92,13 @@ message:
 using your real name (sorry, no pseudonyms or anonymous contributions.) You can
 add the sign off when creating the git commit via `git commit -s`.
 
+If you want signing to be automatic you can set up some aliases:
+
+```
+git config --add alias.amend "commit -s --amend"
+git config --add alias.c "commit -s"
```

Comment:
Do these actually work? I tried once but it didn't help.

Reply:
I will try, if they don't work I will remove them.

Reply:
Does not work for me I will remove it